src/contract/userop/index.ts

```diff
@@ -0,0 +1,86 @@
+import { UserOperationEventEvent } from "userop/dist/typechain/EntryPoint";
+import { VerifiedWallet } from "../../wallet";
+import { ContractInterface, ethers, Signer, utils } from "ethers";
+import { Client, Presets } from "userop";
+
+interface Params {
+    apiKey:string, 
+    paymasterUrl:string,
+    value:string
+}
+
+const response: { transactionHash: string | null, result: UserOperationEventEvent | null } = { transactionHash: null, result: null }
+export type UseropResponse = typeof response;
+
+ class VerifiedContract {
+    private signer: VerifiedWallet | Signer;
+    private contract: ethers.Contract;
+    private abiInterface: ContractInterface;
+    private address: string;
+    constructor(address: string, abi: string, signer: VerifiedWallet | Signer){
+        this.signer = signer;
+        this.abiInterface = new utils.Interface(abi)
+        this.contract = new ethers.Contract(address, this.abiInterface, signer);
+        this.address = address
```

Comment:
- I had to include `this.address` in constructor because I wanted to use it in `callContract` method.
- `this.abiInterface` and `this.contract ` are not used but I never wanted to break SDK contracts by following the convention of `VerifiedContract`